include/xsimd/types/xsimd_batch_constant.hpp

```diff
@@ -287,13 +302,41 @@ namespace xsimd
         return {};
     }
 
+    /**
```

Comment:
This is s auser-facing API, but kit lacks the proper doxygen group and doc integration. I'll fix this.